infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/RoutingQueuesSnapshotReader.java

```diff
@@ -62,9 +58,12 @@ public class RoutingQueuesSnapshotReader extends BaseSnapshotReader {
 
     // Timeout value for which the reader waits for new data to arrive from the Client.  Once the timeout is
     // exceeded, the current snapshot sync gets cancelled and a new one is started all over again.
-    // TODO: The timeout is currently set to 40 mins.  This can be revisited later to determine a more appropriate
+    // TODO: The timeout is currently set to 2s.  This can be revisited later to determine a more appropriate
     //  value.
-    private static final long DATA_WAIT_TIMEOUT_MS = 2400000;
+    @VisibleForTesting
+    public static long DATA_WAIT_TIMEOUT_MS = 120000;
```

Comment:
final

Reply:
Actually, this variable is being set to a lower value in the test...so cannot declare it final